db/init.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Users (
+    id SERIAL PRIMARY KEY,
+    email TEXT UNIQUE,
+    passwordHash TEXT NOT NULL
+);
```

Comment:
```suggestion
CREATE TABLE users (
    id uuid PRIMARY KEY,
    email text UNIQUE,
    password_hash text NOT NULL
);
```
Database should use snake_case. Datatypes ar typically written in lowercase. UUIDs are more ergonomic than integers.